Calmspace/DataHolder.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  DataHolder.swift
+//  Calmspace
+//
+//  Created by Admin on 27/11/2023.
+//
+
+import Foundation
+
+class DataHolder {
```

Comment:
this is a smart idea, to use a global object to store data 🙂 Unfortunately, in real apps, it is often impractical: it is almost certain that many places will read and write the data at the same time, and it results in Data Races and other unpleasant hard to track bugs. 

Try to replace this mechanism by passing data directly from one scene to another. Let me know if it is not 100% clear what I meant here.